Readme.md

```diff
@@ -238,6 +238,33 @@ const prettifyQuery = value => {
 }
 ```
 
+Additionally, `customPrettifiers` can be used to format the time and level outputs:
+```js
+{
+  customPrettifiers: {
+    // The argument for this function will be the same
+    // string that's at the start of the log-line by default:
+    time: timestamp => `🕰 ${timestamp}`,
+
+    // The argument for the level-prettifier may vary depending
+    // on if the levelKey option is used or not.
+    // By default this will be the same numerics as the Pino default:
+    level: logLevel => `LEVEL: ${logLevel}`
+  }
+}
+```
+
+Note that prettifiers do not include any coloring, if the stock coloring on `level` is desired, it can be accomplished using the following:
+```js
+const { colorizerFactory } = require('pino-pretty')
+const levelColorize = colorizerFactory(true)
+const levelPrettifier = logLevel => `LEVEL: ${levelColorize(logLevel)}`
+//...
+{
+  customPrettifiers: { level: levelPrettifier }
+}
+```
```

Comment:
```suggestion
Additionally, `customPrettifiers` can be used to format the time and level
outputs:
```js
{
  customPrettifiers: {
    // The argument for this function will be the same
    // string that's at the start of the log-line by default:
    time: timestamp => `🕰 ${timestamp}`,

    // The argument for the level-prettifier may vary depending
    // on if the levelKey option is used or not.
    // By default this will be the same numerics as the Pino default:
    level: logLevel => `LEVEL: ${logLevel}`
  }
}
```

Note that prettifiers do not include any coloring, if the stock coloring on
`level` is desired, it can be accomplished using the following:
```js
const { colorizerFactory } = require('pino-pretty')
const levelColorize = colorizerFactory(true)
const levelPrettifier = logLevel => `LEVEL: ${levelColorize(logLevel)}`
//...
{
  customPrettifiers: { level: levelPrettifier }
}
```
```

Reply:
@pearofducks somehow the suggestion removed the example. Let's add it back:

```suggestion

Note that prettifiers do not include any coloring, if the stock coloring on
`level` is desired, it can be accomplished using the following:
```js
const { colorizerFactory } = require('pino-pretty')
const levelColorize = colorizerFactory(true)
const levelPrettifier = logLevel => `LEVEL: ${levelColorize(logLevel)}`
//...
{
  customPrettifiers: { level: levelPrettifier }
}
```
```